shared/domains/chelonia/encryptedData.js

```diff
@@ -350,6 +350,7 @@ export const isRawEncryptedData = (data: any): boolean => {
 }
 
 export const unwrapMaybeEncryptedData = (data: any): { encryptionKeyId: string | null, data: any } | void => {
+  if (data == null) return
```

Comment:
This is a 'type' fix, for making `unwrapMaybeEncryptedData` safe to call on `null` | `undefined`.